content/post/087-deprecating-the-old-docs-site.md

```diff
@@ -0,0 +1,53 @@
+---
+date: 2020-04-01
```

Comment:
prooooobably good to avoid april fools day to any real announcements

Reply:
Haha! Yeah good point. Just threw that in there as a placeholder. I'll set it to Friday for now.